studio/src/editors/merch-card-editor.js

```diff
@@ -189,13 +189,21 @@ class MerchCardEditor extends LitElement {
                 <sp-field-label for="prices">Prices</sp-field-label>
                 <rte-field
                     id="prices"
-                    inline
                     data-field="prices"
                     default-link-style="primary-outline"
                     @change="${this.updateFragment}"
-                    >${unsafeHTML(form.prices.values[0])}</rte-field
+                    >${unsafeHTML(form.prices?.values[0])}</rte-field
                 >
             </sp-field-group>
+            <sp-field-label for="promo-text">Promo Text</sp-field-label>
```

Comment:
@3ch023 please use a `sp-field-group` with `toggle` class so that the field toggles on/off based on the metadata of the card variant.
If a card does not support promo text field, it should be hidden automatically.

Reply:
missed that, sorry.
will add now